app/dashboard/package.json

```diff
@@ -15,16 +15,17 @@
   },
   "scripts": {
     "compile": "tsc",
-    "typecheck": "tsc --noEmit",
+    "typecheck": "tsc",
     "build": "vite build",
+    "lint": "eslint .",
     "dev": "vite",
     "dev:e2e": "vite -c vite.test.config.ts",
     "dev:e2e:ci": "vite -c vite.test.config.ts build && vite preview --port 8080 --strictPort",
-    "test": "npm run test:unit && npm run test:e2e",
+    "test": "pnpm run /^test:.*/",
     "test:unit": "vitest run",
-    "test:unit:debug": "vitest",
+    "test-watch:unit": "vitest",
     "test:e2e": "cross-env NODE_ENV=production playwright test",
-    "test:e2e:debug": "cross-env NODE_ENV=production playwright test --ui"
+    "test-watch:e2e": "cross-env NODE_ENV=production playwright test --ui"
```

Comment:
this opens the test runner UI, not a watcher

Reply:
Right. Would a `test-dev` common prefix fit better here? I needed to change those names only because selecting the jobs with regexes is a bit wonky in pnpm, as they match too easily to more complicated patterns (e.g. `/^test:/` regex will also match `dev-test:unit` script, idk why).

Reply:
yeah, sounds good